targets.go

```diff
@@ -71,23 +77,48 @@ func (t interfaceTarget) get() reflect.Value {
 }
 
 func (t interfaceTarget) set(v reflect.Value) error {
-	return t.x.set(v)
+	err := t.x.set(v)
+	if err != nil {
+		return fmt.Errorf("interfaceTarget set: %w", err)
```

Comment:
why wrapper all errors?

Reply:
I think it all goes down to [my comment here](https://github.com/pelletier/go-toml/pull/530#issuecomment-829783833)

Reply:
Let's disable this linter.

After reading the post from that author and Dave Cheney, this is linter seems like the wrong enforcement.